EngineLayer/Proteomics/ProteinExtensions.cs

```diff
@@ -60,84 +60,139 @@ public static IEnumerable<PeptideWithPossibleModifications> Digest(this Protein
                                 while (oneBasedIndicesToCleaveAfter[i] < proteolysisProduct.OneBasedBeginPosition)
                                     i++;
                                 // Start peptide
-                                if (i + missed_cleavages < oneBasedIndicesToCleaveAfter.Count && oneBasedIndicesToCleaveAfter[i + missed_cleavages] <= proteolysisProduct.OneBasedEndPosition && proteolysisProduct.OneBasedBeginPosition.HasValue)
+                                if (i + maximumMissedCleavages < oneBasedIndicesToCleaveAfter.Count && oneBasedIndicesToCleaveAfter[i + maximumMissedCleavages] <= proteolysisProduct.OneBasedEndPosition && proteolysisProduct.OneBasedBeginPosition.HasValue)
```

Comment:
Full should have been left untouched. Change maximumMissedCleavages  back to missed_cleavages